### PR TITLE
UCT/RC: Fix QP destroy for EXP flow

### PR DESCRIPTION
**Pull request by Artemy-Mellanox**

### CODE REVIEW

**Comment by Artemy-Mellanox** [comment]
bot:pipe:retest

File: src/uct/ib/rc/accel/rc_mlx5_ep.c [hunk 1]
```diff
@@ -985,8 +985,13 @@ void uct_rc_mlx5_ep_cleanup_qp(uct_ib_async_event_wait_t *wait_ctx)
...
     uct_ib_mlx5_md_t *md              = ucs_derived_of(iface->super.super.super.md,
                                                        uct_ib_mlx5_md_t);
+    int count;
 
-    uct_rc_mlx5_iface_common_check_cqs_ci(iface, &iface->super.super);
+    count = uct_rc_mlx5_iface_commom_clean(&iface->cq[UCT_IB_DIR_RX],
+                                           &iface->rx.srq,
+                                           ep_cleanup_ctx->qp.qp_num);
+    iface->super.rx.srq.available += count;
+    uct_rc_mlx5_iface_common_update_cqs_ci(iface, &iface->super.super);
```
Comment (yosefe) [inline src/uct/ib/rc/accel/rc_mlx5_ep.c:994]: 1. would it be used in non-exp flow? can we disable?
2. did we already get last wqe reached at this point? need to assert it?

Comment (Artemy-Mellanox) [reply src/uct/ib/rc/accel/rc_mlx5_ep.c:994]: 2. this is either last_wqe callback or iface destroy

Comment (yosefe) [reply src/uct/ib/rc/accel/rc_mlx5_ep.c:994]: 2. does it make sense to add some assert the last wqe event arrived?

Comment (Artemy-Mellanox) [reply src/uct/ib/rc/accel/rc_mlx5_ep.c:995]: 2. by design we get here either in response to last_wqe or if we don't care of it anymore so assert won't work here,

**Review by yosefe — APPROVED** [review]

**Comment by yosefe** [comment]
@Artemy-Mellanox pls squash

**Comment by pentschev** [comment]
Could we backport this to v1.10.x? This is certainly a blocker for any use of MOFED 4 in 1.10.

**Comment by yosefe** [comment]
> Could we backport this to v1.10.x? This is certainly a blocker for any use of MOFED 4 in 1.10.

@Artemy-Mellanox can u pls backport?

**Pull request merged**